src/resolve.cxx

```diff
@@ -731,6 +697,85 @@ void makeNewToken(std::string& working, uint64_t i, std::vector<token>& tokens,t
 
     working = "";
 }
+
+static void resolve_I(variable*& left, token& __leftHand, token& t)
```

Comment:
this resolve_I/resolve_II situation sucks but im not sure what else to do about it, so it's staying for now